### PR TITLE
Adds data.gc.ca, Issue #25

### PR DESCRIPTION
**Pull request by thriuin**

### CODE REVIEW

**Comment by johnmartin** [comment]
@thriuin Your screengrab is the incorrect height (not your problem as the guideline dimensions within the example issue were incorrect #1, I've updated that issue now) but would you mind updating the screengrab to 350x250?

File: config/instances.json [hunk 1]
```diff
@@ -554,5 +554,16 @@
 			{ "key": "Region", "value": "Europe" },
 			{ "key": "Type", "value": "Local / Regional Government" }
 		]
+	},
+	{
+		"id": "data_gc_ca",
+		"url": "http://data.gc.ca",
+		"title": "Data.gc.ca",
+		"description": "Canada's new portal, data,gc,ca, is a key part of Canada’s Action Plan on Open Government to enhance transparency and accountability. data.gc.ca provides one-stop access to Government of Canada data and information.",
```
Comment (johnmartin) [inline config/instances.json]: `data,gc,ca` should be `data.gc.ca` ?

**Comment by johnmartin** [comment]
@thriuin Fix these and I'll happily merge away.

**Comment by thriuin** [comment]
@johnmartin. Thanks for catching the typo - it was definitely incorrect. I have also updated the screenshot to  350x250.

Thanks again

**Pull request merged**